core/src/main/java/com/scalar/db/transaction/consensuscommit/CommitHandlerWithGroupCommit.java

```diff
@@ -84,7 +84,10 @@ public void commitState(Snapshot snapshot)
     commitStateViaGroupCommit(snapshot);
   }
 
-  private void groupCommitState(String parentId, List<Snapshot> snapshots)
+  // TODO: Emitter interface should have `emitNormalGroup()` and `emitDelayedGroup()` separately and
```

Comment:
Found there was room to improve this method and related ones to make which type of ID is passed clearer, so added this TODO.